active/003-seq-feature.md

```diff
@@ -0,0 +1,108 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
```

Comment:
I am fine with the naming, but better naming is always welcome : D

On Fri, Apr 29, 2016 at 10:40 AM, Jai Ram Rideout <notifications@github.com>
wrote:

> In active/003-seq-feature.md
> <https://github.com/biocore/scikit-bio-rfcs/pull/9#discussion_r61617260>:
>
> > @@ -0,0 +1,108 @@
> > +---
> > +Feature name: interval-metadata
> > +Start date: <2015-12-18>
> > +Pull request: 1
> > +Authors: ["@RNAer", "@mortonjt"]
> > +Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
> > +---
> > +
> > +# Summary
> > +
> > +Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
>
> Cool! @RNAer <https://github.com/RNAer> @ebolyen
> <https://github.com/ebolyen> @wasade <https://github.com/wasade>
> @gregcaporaso <https://github.com/gregcaporaso> (and anyone else), I'm
> really interested in your opinion on naming here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio-rfcs/pull/9/files/a7c4b661d4f1f08e309222e0245d4394a27c8767#r61617260>
>